apps/src/templates/studioHomepages/SetUpMessage.jsx

```diff
@@ -74,11 +59,11 @@ const SetUpMessage = ({
   onClick,
 }) => (
   <div style={styles.section} className={className}>
-    <div style={styles.wordBox}>
-      <div style={isRtl ? styles.rtlHeading : styles.heading}>
+    <div style={[styles.wordBox, isRtl ? styles.rtl : styles.ltr, {padding:0}]}>
```

Comment:
Even without hoisting out of React, you could do something like `const localeStyle = isRtl ? styles.rtl : styles.ltr` at the top of your render function, and then replace all these instances with `localeStyle`